### DISCUSSION TITLE
thumbnails scroll

### DISCUSSION THREAD

**Discussion by MyNameTony1000** [fancyapps/ui#513] [Q&A]
Hello, I'm using the (https://stackblitz.com/edit/js-2k1udh?file=index.html,index.js,style.css) example and I wanted to know if it's possible to somehow change the slide transition when using the mouse wheel to scroll the thumbnail block?

**Comment by fancyapps** [comment]
Hi,

You are referring to the thumbnails inside the Fancybox, right? The transition is the same as the default, you can adjust it like this:

```js
Fancybox.bind('[data-fancybox="gallery"]', {
  Carousel: {
    transition: 'slide', 
  },
});
```

Now, carousel inside Fancybox will animate with "slide" transition when user clicks new/prev arrow or uses mouse wheel over the thumbnails.

**Reply by MyNameTony1000** [reply]
I meant a little different. When you try to scroll through the thumbnails, the block will not scroll, but the slides will simply switch. I asked about the ability to scroll thumbnails with the mouse wheel. Like regular blocks width overflow: scroll

**Reply by fancyapps** [reply]
Oh, in that case, sorry, there is currently no option for. You could try to create your own plugin.

**Reply by MyNameTony1000** [reply]
Thanks for the quick response))
